Linux_scripts/RHUI_repo_validation_scripts/repo_check.py

```diff
@@ -1,40 +1,53 @@
 import os
+import sys
+import logging
+from logging.handlers import RotatingFileHandler
+from logging import handlers
 import subprocess
 import platform
 from threading import Timer
 import socket
 import time
 from scenarios import *
 
+
 ########### Function for RHEL VMs ##############
 def redhat():
       ##### checking connectivity of RHEL Repo server IPs #####
 
    def ip_tables_check():
-      reject_rule = subprocess.Popen("iptables -L OUTPUT -v -n | grep -i reject", stdout=subprocess.PIPE, shell=True)
+      reject_rule = subprocess.Popen("iptables -L OUTPUT -v -n | egrep -i 'reject|drop'", stdout=subprocess.PIPE, shell=True)
       rej_rule = str(reject_rule.communicate()[0])
-      if "tcp dpt:443 reject-with icmp-port-unreachable" in rej_rule or "tcp dpt:https reject-with icmp-port-unreachable" in rej_rule or "   reject-with icmp-port-unreachable" in rej_rule:
-         print(rej_rule)
+      if "tcp dpt:443 reject-with icmp-port-unreachable" in rej_rule or "tcp dpt:https reject-with icmp-port-unreachable" in rej_rule or "   reject-with icmp-port-unreachable" in rej_rule or "tcp dpt:443" in rej_rule:
```

Comment:
make these commands as constants and add them to a list,
iterate that list to check for matching condition

Reply:
Done